views/restaurant-home.html

```diff
@@ -0,0 +1,429 @@
+                        <div class="modal-container card-py modal-width-80">
+                            <div class="flex flex-between  flex-align-item-center pt-1 pb-1">
+                                <h1 class="font-mid-bold card-header-text">Modal Header</h1>
+                                <span
+                                    class="material-icons  card-close-btn flex flex-justify-center flex-align-item-center"
+                                    id="modalCloseBtn">close</span>
+                            </div>
+                            <div class="card-mb-1 model-content py-1">
+                                <div class="tabs flex flex-wrap">
+                                    <input type="radio" class="tabs-radio" name="tabs-group" id="tabs-group1" checked>
+                                    <label for="tabs-group1" class="tabs-title">Features</label>
+                                    <div class="tabs-content">
+                                        hi1
+                                    </div>
+                                    <input type="radio" class="tabs-radio" name="tabs-group" id="tabs-group2">
+                                    <label for="tabs-group2" class="tabs-title">Features</label>
+                                    <div class="tabs-content">
+                                        hi2
+                                    </div>
+                        
+                                    <input type="radio" class="tabs-radio" name="tabs-group" id="tabs-group3">
+                                    <label for="tabs-group3" class="tabs-title">Features</label>
+                                    <div class="tabs-content">
+                                        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Praesentium repudiandae eius voluptates iusto
+                                        nobis vel, cupiditate doloribus adipisci consectetur delectus dolorum quod repellendus soluta? Qui
+                                        aperiam laudantium enim corrupti laboriosam!
+                                    </div>
+                        
+                                    <input type="radio" class="tabs-radio" name="tabs-group" id="tabs-group4">
+                                    <label for="tabs-group4" class="tabs-title">Features</label>
+                                    <div class="tabs-content">
+                                       Lorem ipsum, dolor sit amet consectetur adipisicing elit. Culpa, nam nobis, impedit ipsam assumenda dolores molestias delectus cumque corporis enim dolorem incidunt nihil similique sint libero amet iste exercitationem eaque.
+                                    </div>
+                        
+                                   
+                                </div>
+                    
+                            </div>
+                            <div class="flex  pt-1 pb-1 flex-justify-end">
```

Comment:
class names can be more descriptive here

Reply:
> class names can be more descriptive here

@iRohitGaur here pt-1 class describes padding-top and pb-1 padding bottom